.github/workflows/nightly_tests.yaml

```diff
@@ -89,6 +89,14 @@ jobs:
       run : python3 xpk.py info --cluster $TPU_CLUSTER_NAME --zone=us-central2-b
     - name: Delete the workload on the cluster
       run: python3 xpk.py workload delete --workload $WORKLOAD_NAME --cluster $TPU_CLUSTER_NAME --zone=us-central2-b
+    - name: Run a batch job on the cluster
+      run: python3 xpk.py batch --cluster $TPU_CLUSTER_NAME --zone=us-central2-b test.sh
+    - name: List out the jobs on the cluster
+      run: python3 xpk.py job ls | grep 'xpk-def-app-profile-slurm-'
```

Comment:
are we checking for presence of job we ran?

Reply:
In run 113 we are getting the last job started, which - if the task "Run a batch job on the cluster" completed successfully - should be the batch job.